### PR TITLE
Follow-up to Alternate bindPassword/clientSecret methods (PR#1669)

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/1669

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@tnguyen-rh @bfallonf @tpoitras @adellape PTAL

**Comment by ahardin-rh** [comment]
Changes also satisfy https://github.com/openshift/openshift-docs/pull/1621

File: install_config/master_node_configuration.adoc [hunk 1]
```diff
@@ -345,9 +345,13 @@ with the 'gquota' option, and the matching security context contraint's fsGroup
...
 
-=== Parallel image pulls with Docker 1.9+
+=== Parallel Image Pulls with Docker 1.9+
 
-If you are using Docker 1.9+, you may want to consider enabling parallel image pulling, as the default is to pull images one at a time. Please note that there is a potential issue with data corruption prior to Docker 1.9, but starting with 1.9, the corruption issue has been resolved, so it is safe to switch to parallel pulls.
+If you are using Docker 1.9+, you may want to consider enabling parallel image
+pulling, as the default is to pull images one at a time. Please note that there
+is a potential issue with data corruption prior to Docker 1.9. However, starting
+with 1.9, the corruption issue is resolved and it is safe to switch to
+parallel pulls.
```
Comment (bfallonf) [inline install_config/master_node_configuration.adoc]: @ahardin-rh Would the "Please note..." stuff be better in a note box?

**Comment by bfallonf** [comment]
@ahardin-rh Just one question from me. But otherwise, looks good.

**Comment by ahardin-rh** [comment]
[rev_history]
|link:../install_config/master_node_configuration.html[Master and Node Configuration]
|Added information about link:../install_config/master_node_configuration.html#master-node-configuration-parallel-image-pulls-with-docker[parallel pulls with Docker 1.9+].
%
|link:../install_config/master_node_configuration.html[Master and Node Configuration]
|Added information about alternate bindPassword/clientSecret methods.
%
|link:../iinstall_config/syncing_groups_with_ldap.html[Syncing Groups With LDAP]
|Added information about alternate bindPassword/clientSecret methods.

**Pull request merged**